keras_rcnn/layers/object_detection/object_proposal.py

```diff
@@ -0,0 +1,59 @@
+import keras.backend
```

Comment:
Ah. I misunderstood. This is a style convention:

https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles